Rakefile

```diff
@@ -4,5 +4,20 @@ require 'middleman-gh-pages'
 require 'rspec/core/rake_task'
 RSpec::Core::RakeTask.new(:spec)
 
+desc "Compile javascript -> coffeescript"
+task :coffee do
+  destination = "source/javascripts/refills/coffeescript"
+  Dir["source/javascripts/refills/**/*.js"].each do |source|
+    filename = File.basename(source.gsub(%r{.js}, ".coffee"))
+
+    `js2coffee #{source} > #{destination}/#{filename}`
+  end
+
+  puts "Please make sure that all generated coffeescript files are correct"
```

Comment:
I am a bit sceptical about just auto committing the generated files, so I just wanted to give an extra warning to look over the generated files manually. But that can absolutely be clarified.
Thoughts?